src/ruby/yast/y2start_helpers.rb

```diff
@@ -137,7 +137,13 @@ def self.generate_exit_code(value)
     private_class_method def self.signal_handler(name)
       Signal.trap(name, "IGNORE")
 
-      $stderr.puts "YaST got signal #{name}."
+      # Exception swallowing: writing to stderr could fail if the parent process was killed,
+      # see bsc#1154854.
+      begin
+        $stderr.puts "YaST got signal #{name}."
```

Comment:
is in that case io already closed? If so, we can even do it without exception with `$stderr.puts "YaST got signal #{name}." unless $stderr.closed?`

Reply:
Nop, is not closed :(, at least the method returns *false*.

Reply:
so maybe mention it in comment? I expect I am not alone who ask